Sources/PackageModel/UserToolchain.swift

```diff
@@ -741,7 +741,11 @@ public final class UserToolchain: Toolchain {
             // targetInfo from the compiler
             let targetInfo = try customTargetInfo ?? Self.getTargetInfo(swiftCompiler: swiftCompilers.compile)
             self._targetInfo = targetInfo
-            triple = try swiftSDK.targetTriple ?? Self.getHostTriple(targetInfo: targetInfo, versioned: false)
+            var defaultTriple = try Self.getHostTriple(targetInfo: targetInfo, versioned: false)
+            if defaultTriple.isOpenBSD() {
+                defaultTriple = try Self.getHostTriple(targetInfo: targetInfo, versioned: true)
+            }
+            triple = swiftSDK.targetTriple ?? defaultTriple
```

Comment:
This was already the case before your patch, but swiftSDK.targetTriple should always be nil if we're in this else branch

Reply:
Makes sense; I've elided this.